pyproject.toml

```diff
@@ -12,6 +21,11 @@ pre-commit = "^2.13.0"
 volttron-client = "^0.3.8"
 openleadr = "^0.5.24"
 
+[tool.poetry.scripts]
+volttron_openadr_ven="volttron_openadr_ven.agent:main"
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/VOLTTRON/volttron-openadr-ven/issues"
```

Comment:
please add pipy classifiers and the python versions that we are utilizing

https://pypi.org/classifiers/

Reply:
I think we can say this is either an alpha or beta development classifier.

Reply:
Will do. I will use [this example from volttron-client](https://github.com/VOLTTRON/volttron-client/blob/main/setup.cfg) as a guide for the classifier section.